src/shared_modules/content_manager/src/components/CtiDownloader.hpp

```diff
@@ -109,9 +109,10 @@ class CtiDownloader : public AbstractHandler<std::shared_ptr<UpdaterContext>>
      * @brief Get the CTI API base parameters.
      *
      * @param ctiURL Base URL from where to download the CTI parameters.
+     * @param userAgent HTTP user agent.
      * @return struct CtiBaseParameters Base parameters of the CTI API.
      */
-    CtiBaseParameters getCtiBaseParameters(const std::string& ctiURL)
+    CtiBaseParameters getCtiBaseParameters(const std::string& ctiURL, const std::string& userAgent)
```

Comment:
Question: would it make sense to set the userAgent as a member variable of these classes (APIDownloader, CtiDownloader, ...), and set its value in the constructor?
Just to keep the interface of these other functions (like getCtiBaseParameters) simpler.

Reply:
I agree. If there is no reason for changing the user agent parameter on every method call, then it would be better to set this parameter at the constructor of each class.

Reply:
Yes, in fact, I realized the context is already present here, so no need for additional parameters/members :sweat_smile: